### ISSUE TITLE
Not all system icons are used on kde

### ISSUE THREAD

**Issue by qlum** [jonaski/strawberry#276]
**Describe the bug**
When using the "Use system theme icons" option not all applicable system icons are used.
Most notably play / pause / stop. 
Looking through the interface I found the following icons that could have been used but are not though there are probably more:
The play / pause / stop icons are not used at all. 
open audio cd (cd-rom icon)
equalizer: equalizer icon
under devices non of the device icons are used where there are quite a few.
under settings > backend the con for audio card could be used
settings > shortcuts the keyboard icon could be used. 
settings > proxy the proxy icon could be used instead of the general network icon

**Screenshot**
![Screenshot_20191004_115341](https://user-images.githubusercontent.com/3518815/66201663-24ec4600-e6a4-11e9-91c3-85067eaaeb8d.png)
Please note that the top icon bar is of clementine^

![Screenshot_20191004_124109](https://user-images.githubusercontent.com/3518815/66201721-40575100-e6a4-11e9-984d-5c83f315458e.png)

**System Information:**
 - Operating system: ArchLinux (kde + breeze icon theme)
 - Strawberry Version: 0.65

**Additional context**
From what I have seen clementine does it better on this regard and at least uses the proper play / pause / stop icons.

Please note that I am not very practiced at github reports.

**Comment by jonaski** [comment]
I have removed the option from the settings. It caused problems with the UI when incorrect icons are loaded, so it doesn't work correctly with all themes and causes additional issues.
It's a lot of work to test it access different desktops and installed themes, and the names often aren't standardized either. So far too few contributors to prioritize it.
From: https://developer.gnome.org/icon-naming-spec/

> The Icon Theme Specification has been in use now for a while, in several desktops, including KDE and Gnome. However, there has never been any centralized direction on how to name the icons that are available for use by applications, when creating a theme. This has meant that artists have historically had to duplicate many icons, in order for their themes to work across desktop environments.

**Comment by mshmm** [comment]
That's a shame as it works very well with the Papirus icon theme which I use.

**Comment by mshmm** [comment]
Could the feature perhaps be retained with a warning when selected? (E.g. selecting the "Use x11's shortcut keys" generates a warning under KDE). The "Use system icon theme" feature is not enabled by default and works well for some.

**Comment by mshmm** [comment]
In fact, Papirus has had icons added to the theme specifically for Strawberry:
[https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/issues/1723](url)
[https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/commit/342250d393c2f86255cad2b7dfde23a8ac00ca4d](url)
[https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/commit/ef8a85a6ea8910a362c0dfb820a49ac47aecbdbc](url)
[https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/commit/28e1d120b40ecb2867401181b9e9e325159b3a93](url)

Please restore the option to use this icon theme in Strawberry.

**Comment by jonaski** [comment]
I will look into if it can be enabled again before next release.
Need to figure out how to avoid things like https://github.com/jonaski/strawberry/issues/268

**Comment by mshmm** [comment]
Thank you. #268 could be theme related, and the bug report doesn't say what desktop and GTK (or KDE) theme is being used.

**Comment by jonaski** [comment]
I have renamed some of the playback icons and cd to media-optical. For some others I have added alternative names it can fallback to in case the first one doesn't match. We could probably add more device icons too.
The alternative icon names can be added to a iconmapper, so that multiple names can be added to better work with different themes. The file is here: https://github.com/jonaski/strawberry/blob/master/src/core/iconmapper.h
Re-enabled the system theme icons option.
The plan is also to set min / max size and do size checks so icons with unsuitable sizes are not loaded, I have not completed that yet.
